src/core/gridSystem.js

```diff
@@ -70,6 +70,9 @@ module.exports = function gridSystem(options) {
                                if(lng.indexOf(".") != -1) lng = lng.split('.')[0] + '.' + lng.split('.')[1].slice(0,decimalPlacesAfterZero)
                                return '' + lng + 'W';
                            }
+                           else if(lng == 0) {
```

Comment:
Actually where lng was 0 it did not return anything, so undefined was displayed, to correct that I added this, also we did not need N or S with it so it had to be separate.